src/components/Navbar/constants.tsx

```diff
@@ -8,56 +7,45 @@ type ResourceItem = {
 
 export const resourcesMenuItems = (mobile?: boolean): ResourceItem[] => {
   const linkStyle = {
-    color: 'var(--text-primary)',
-    fontWeight: mobile ? 400 : 500,
+    className: 'nav-dropdown-item',
+    target: '_blank',
+    rel: 'noopener noreferrer',
+    style: {
+      color: 'var(--text-primary)',
+      fontWeight: mobile ? 400 : 500,
+    },
   }
   return [
     {
       key: 'governance',
       label: (
-        <ExternalLink
-          href="https://vote.juicebox.money/#/jbdao.eth"
-          className="nav-dropdown-item"
-          style={linkStyle}
-        >
+        <a href="https://vote.juicebox.money/#/jbdao.eth" {...linkStyle}>
```

Comment:
**question** any reason not to use ExternalLink here?

Reply:
I thought it would be a little easier to grok having all of the props co-located in one place since the items array is eventually going to live in its own file it seemed like co-locating the props made sense.

Also, it's a bit more explicit, we know 100% that we're using an `a` tag and not `next/link`